README.md

```diff
@@ -124,3 +128,71 @@ interpreting them with HIBA.
 libhiba provides convenience functions to parse and extract HIBA extensions
 directly from a certificate, but it does not verify the certificate. It assumes
 the certificate was validated by a prior stage.
+
+### Example CA
+
+Once you have completed the
+[compilation instructions](https://github.com/google/hiba/blob/main/INSTALL.md#compilation)
+you will have a `hiba-gen` and `hiba-ca.sh` in your hiba directory. You can now
+create CA keys, grants, identities, grant eligibility, and sign host and user
+certificates with these extensions. The following assumes you are in your hiba
+directory.
+
+1. Choose a directory for the CA structure.
+  * `$ export dest=/tmp/hiba-ca`
+1. Generate a CA keypair.
+  * ```
+$ ./hiba-ca.sh -c -d "$dest" -- -N ""
+== Generating CA keys in /tmp/hiba-ca/ca
+== Done ==
+```
+1. Generate a test user keypair (alternatively you can copy existing keys to
```

Comment:
You can't just copy keys, but you can import a user existing pubkey (no need for the private key) by running:

hiba-ca.sh -i -u -f path_to_pubkey -I principal

Reply:
updated - first in this file but I'm going to move everything to its own md file and out of readme.